crates/pop-cli/src/commands/call/parachain.rs

```diff
@@ -191,12 +194,21 @@ impl CallParachainCommand {
 					cli.input("Signer of the extrinsic:").default_input(DEFAULT_URI).interact()?,
 			};
 
+			// Prompt the user to confirm if they want to execute the call with sudo privileges.
+			if !self.sudo {
+				self.sudo = cli
+					.confirm("Would you like to execute this operation with sudo privileges?")
```

Comment:
```suggestion
					.confirm("Would you like to dispatch this function call with `Root` origin?")
```
Based on the parameter description above and the docs at https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame/sudo

Reply:
Perhaps one for the future, but this prompt only makes sense if the sudo pallet is mentioned in the chain metadata. There is also an assumption here that it is named 'Sudo', when in fact any chain could have chosen a different name.

Reply:
Good catch.
It makes sense to check if Sudo exist before prompt the user, added in this commit https://github.com/r0gue-io/pop-cli/pull/349/commits/935744ac60b0838d24f53362d40c34437e2f2bb5

However it is making the assumption that it is named 'Sudo', but not sure how we can improve this.

Reply:
Likewise. I like that you check the dispatchable name too!